### ISSUE TITLE
Error: Unknown error in Prisma Client

### ISSUE THREAD

**Issue by thesunny** [prisma/prisma-client-js#723]
## Bug description

I get this error when I refresh a web page that in the back end makes Prisma calls. This issue happens on Heroku. It does not happen on my local Mac development server.

On Heroku, it's unpredictable (ie. doesn't happen on every call) but it happens consistently (ie. it will always happen if we give it enough calls).

```
Error: Unknown error in Prisma Client

This is a non-recoverable error which probably happens when the Prisma Query Engine has a panic.
```

One pattern is that if the number of Prisma calls is few on a page, it is less likely to happen. I have to refresh the page a lot. If there are a lot of calls, I can get it to happen on most calls.

## How to reproduce

Unfortunately, not sure how to make a minimal reproduction for this; however, I'd appreciate feedback on how to narrow it down.

One thing I am uncertain of is whether I can make multiple calls through the same Prisma Client, even while another Prisma call is waiting. I do make many calls in parallel so I wonder if I need to create a separate Prisma Client for each call or have to wait until earlier calls are done? I assumed that there is a connection manager that handled this for me but I haven't checked.

## Expected behavior

Doesn't crash

## Prisma information

```
generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["native", "darwin", "debian-openssl-1.1.x"]
}

datasource db {
  provider = "postgresql"
  url      = env("HEROKU_POSTGRESQL_PINK_URL")
}
```

I didn't provide the db schemas since it doesn't seem to matter what they are. It happens on any page regardless of what the Prisma calls are.

## Environment & setup

- Database: PostgreSQL

| Name     | Version            |
|----------|--------------------|
| Node     | v12.16.2           | 
| OS       | debian-openssl-1.1.x|
| Prisma   | 2.0.0-beta.8       |

Note: Works fine on OSX with same code

**Comment by thesunny** [comment]
I found the reason for this error. It may be worth documenting.

The way I used the PrismaClient is that I created an instance for each connection to the API server. So when a request was made, an instance was created. When the request was finished, I closed the connection thinking I was being a good citizen.

This caused what was likely some race conditions.

I think it would be useful to document how the PrismaClient should be used. In the example for RPC, I see that a PrismaClient instance is created once for the entire app. I switched to this and the errors went away.
